app/views/users/show.html.erb

```diff
@@ -1,9 +1,9 @@
 <h2>User shows</h2>
 
-<% if !@users.kind_of? String %>
-    <h4><%= @users.name %></h4>
-    <p><%= @users.bio %></p>
-    <p><%= @users.id %></p>
+<% if !@user.kind_of? String %>
+    <h4><%= @user.name %></h4>
+    <p><%= @user.bio %></p>
+    <p><%= @user.id %></p>
 <% else %>
-    <p><%= @users %>
-<% end %>
+    <p><%= @user %>
```

Comment:
- [ ] Kindly ensure you use the `#recent_posts` method to render only the user's 3 most recent post.

- [ ] @SekoViper great work on your app however the design of this page doesn't match the wireframe given, below is a comparison for better understanding:


Wireframe Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/225416214-5474cb6c-197a-4500-a29b-115b05f5faef.png) |  ![image](https://user-images.githubusercontent.com/61408860/225416279-e5f262c0-57cb-4b80-92a4-8fa0c68eed5b.png)